### ISSUE TITLE
Should there be a couple extra steps in the "Getting Started"?

### ISSUE THREAD

**Issue by johnnyreilly** [TypeStrong/atom-typescript#131]
For my part until I did 3 and 4 below I had no joy at all:

1. Install atom.
2. `apm install typescript` (apm needs git in your path)
3. Fire up atom, goto Settings / Packages and click "install" on atom-typescript
4. Restart atom 

Happy to submit PR if useful.

**Comment by basarat** [comment]
Perhaps 

1. Install atom.
2. `apm install typescript` (apm needs git in your path)
3. Fire up atom. Wait for the message `Some dependent packages were required for atom-typescript. These are now installed. Best you restart atom just this once.`
4. Restart atom 

Feel free to commit on master. Members of DT are members of TypeStrong :heart:

**Comment by johnnyreilly** [comment]
Thanks chap!
I was too impatient to hang around for that message so didn't see it. If that's the behaviour experienced generally then happy to change it to that instead.

I think I'll uninstall and reinstall to ensure that is the experience. I've seen this post which suggests I'm not the only one who had had a strange install:
Http://codebetter.com/derikwhittaker/2015/02/07/typescript-support-in-atom-editor-for-windows/

**Comment by johnnyreilly** [comment]
Oh and a question: is `apm install typescript` equivalent to `apm install a to me typescripts?

If I go to https://atom.io/packages/typescript I'm redirected to https://atom.io/packages/atom-typescript which suggests that's the case. (I thought I was maybe attempting to install typescript itself and was wondering if it would end up fighting with the already installed version)

**Comment by basarat** [comment]
Yes both atom-typescript and typescript are equivalent for apm.

**Comment by johnnyreilly** [comment]
Hey @Basarat,

I performed a clean install of Atom so I could be sure what the startup experience was.  Before I get to that I wanted to share a little friction I experienced when I was trying to clean my machine.  I entered:

```
PS C:\Users\john.reilly> apm uninstall typescript
Uninstalling typescript done
```

To uninstall `atom-typescript`. You'd think from the message above it was all good.  Not so!  When I do `apm list` afterwards I get this:

```
C:\Users\john.reilly\.atom\packages (4)
├── atom-typescript@0.71.0
├── autocomplete-plus@2.2.0
├── linter@0.12.0
└── project-manager@1.15.5
```

I only get to remove `atom-typescript` successfully if  I do this:

```
PS C:\Users\john.reilly> apm uninstall atom-typescript
Uninstalling atom-typescript done
```

`apm list` then leaves me with: 

```
C:\Users\john.reilly\.atom\packages (3)
├── autocomplete-plus@2.2.0
├── linter@0.12.0
└── project-manager@1.15.5
```

ie Uninstall is successful on this occasion.  This may be a bug with apm - I may raise an issue for them.

Because of the confusion around this I'd suggest that it might be clearer to get the users to specify `apm install atom-typescript`.  What do you think?

That being the case I'd suggest the instructions could read:

1. Install atom.
2. `apm install atom-typescript`  (apm needs git in your path)
3. Fire up atom. Wait for the message: `Some dependent packages were required for atom-typescript. These are now installed. Best you restart atom just this once.` **It may take up to 5 minutes for this message to appear.  Be patient!**
4. Restart atom

**Comment by basarat** [comment]
+100  apm install typescript did annoy me. Atom-TypeScript it is ❤️

**Comment by basarat** [comment]
👍 all the other text as well

**Comment by johnnyreilly** [comment]
Resolved with 0ab622b06dfcf58f5e9f45b986dff83b82a53041 :heart:
